API/config/README.md

```diff
@@ -0,0 +1,52 @@
+config API
+==========
+
+#### `config.get`
+
+> Returns the currently being used config. If the daemon is off, it returns the stored config.
+
+##### `Go` **WIP**
+
+##### `JavaScript` - ipfs.config.get([key, callback])
+
+`key` is the key of the value that should be fetched from the config file. If no key is passed, then the whole config should be returned. `key` should be of type String.
```

Comment:
Can `key` be in the form of `my.config.value`? as far as I know that is available in go